test/relations.test.js

```diff
@@ -748,7 +793,12 @@ describe('relations', function() {
             should.not.exist(err);
             should.exist(ch);
             ch.should.have.lengthOf(1);
-            ch[0].name.should.eql('a');
+            if (isCassandraConnector) {
+              // Sorting is not set in define for Cassandra
+              ['a', 'c', 'z'].should.containEql(ch[0].name);
```

Comment:
Please consider using `oneOf` instead, it makes the assertion much easier to understand to me.

```js
ch[0].name.should.be.oneOf(['a', 'c', 'z']);
```

https://shouldjs.github.io/#assertion-oneof